badgecheck/tasks/utils.py

```diff
@@ -10,3 +10,14 @@ def task_result(success=True, message='', actions=None):
         actions = []
 
     return (success, message, actions,)
+
+
+def is_empty_list(value):
+    return isinstance(value, (tuple, list,)) and len(value) == 0
+
+
+def abbreviate_value(value):
+    if len(str(value)) < 48:
+        return str(value)
+    else:
+        return str(value)[:48] + '...'
```

Comment:
When you abbreviate, does the abbreviation itself have to follow the abbrev rules? In which case you want `str(value)[:45] + '...'` maybe? (If you don't do this, do you risk an endlessly abbreviating thing that could just swallow up its contents in dot characters if called repeatedly?)